crates/fluvio-controlplane-metadata/src/tableformat/spec.rs

```diff
@@ -30,10 +30,56 @@ impl Default for InputFormat {
         Self::JSON
     }
 }
-//pub struct TableFormatColumnConfig {
-//    pub label: String,
-//    pub width: String,
-//    pub alignment: String, // Can I enum this?
-//    pub path: String,
-//    pub format: String,
-//}
+#[derive(Encoder, Decoder, Default, Debug, PartialEq, Clone)]
+#[cfg_attr(
+    feature = "use_serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(rename_all = "camelCase")
+)]
+pub struct TableFormatColumnConfig {
```

Comment:
I think that one of these fields should be required, not sure which one. I marked them as option because in the CRD they are not required.

What do you think? @tjtelan

Reply:
The `key_path` is the only field that needs to be required